opencti-platform/opencti-graphql/src/database/entity-representative.js

```diff
@@ -83,8 +83,8 @@ export const extractEntityRepresentativeName = (entityData) => {
     mainValue = entityData.description;
   }
   // If no representative value found, return the standard id
-  if (isEmptyField(mainValue) || mainValue === 'Unknown') {
-    return entityData.standard_id;
+  if (isEmptyField(mainValue)) {
+    return 'Unknown';
```

Comment:
so we want to return "Unknown" instead of standard_id ?

Reply:
Not really "instead of", but we couldn't find any logic behind that choice of setting the id rather than "Unknown" when we don't know the representative.
Neither Romuald (you wrote the code) nor Julien could remember a reason, so for now I suggest we do what seems to be coherent with the front : hence "Unknown"